pkg/products/rhssocommon/reconciler.go

```diff
@@ -361,8 +361,11 @@ func (r *Reconciler) ReconcileCloudResources(dbPRefix string, defaultNamespace s
 		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("failed to reconcile database credentials secret while provisioning %s: %w", ssoType, err)
 	}
 
+
 	// at this point it should be ok to create the failed alert.
 	if postgres != nil {
+		// add the applyImmediately flag to the CR
```

Comment:
@austincunningham i think the update for the sso postgres will have to go into the modifyfunction also called by the croutil.ReconcilePostgres function which happens here -> https://github.com/integr8ly/integreatly-operator/blob/master/pkg/resources/rhsso.go#L36